docs/config/README.md

```diff
@@ -347,13 +347,14 @@ discharging_symbol = "💀"
 
 ### Battery Display
 
-The `display` configuration option is used to define when the battery indicator should be shown (threshold) and what it looks like (style).
+The `display` configuration option is used to define when the battery indicator should be shown (threshold) and what it looks like (style). The shown symbol can be specified as (symbol).
 If no `display` is provided. The default is as shown:
 
 ```toml
 [[battery.display]]
 threshold = 10
 style = "bold red"
+symbol = None
 ```
 
 #### Options
```

Comment:
```suggestion
The default value for the `symbol` option is the value of `battery`'s `discharging_symbol` option.

#### Options
```

Reply:
I added the description like your above comment.